xml/art_admin_slemicro.xml

```diff
@@ -1332,11 +1327,771 @@ After=etcd.service
+         uses. The default value is <emphasis>60s</emphasis>. Keep in mind that
+         this value can be overriden by the <command>pmlogger</command> command
+         line.
+        </para>
+       </listitem>
+      </varlistentry>
+      <varlistentry>
+       <term>PMLOGGER_CHECK_SKIP_LOGCONF</term>
+       <listitem>
+        <para>
+         Setting this option to <emphasis>yes</emphasis> disables regeneration
+         and checking of the <command>pmlogger</command> configuration if the
+         configuration <command>pmlogger</command> comes from
+         <command>pmlogconf</command>. The default behavior is to regenerate
+         configuration files and check for changes every time
+         <command>pmlogger</command> is started.
+        </para>
+       </listitem>
+      </varlistentry>
+     </variablelist>
+     <para>
+      An example follows:
+     </para>
+<screen>
+PMLOGGER_LOCAL=1
+PMLOGGER_MAXPENDING=5
+PMLOGGER_INTERVAL=10
+PMLOGGER_CHECK_SKIP_LOGCONF=yes
+     </screen>
+    </section>
+    <section xml:id="sec-control-local">
+     <title>The <filename>/etc/pcp/pmlogger/control.d/local</filename> file</title>
+     <para>
+      The file <filename>/etc/pcp/pmlogger/control.d/local</filename> stores
+      specifications of the host, which metrics should be logged, the logging
+      frequency (default is 24 hours), and <command>pmlogger</command> options.
+      An example follows:
+     </para>
+<screen>
+# === VARIABLE ASSIGNMENTS ===
+#
+# DO NOT REMOVE OR EDIT THE FOLLOWING LINE
+$version=1.1
+
+# Uncomment one of the lines below to enable/disable compression behaviour
+# that is different to the pmlogger_daily default.
+# Value is days before compressing archives, 0 is immediate compression,
+# "never" or "forever" suppresses compression.
+#
+#$PCP_COMPRESSAFTER=0 
+#$PCP_COMPRESSAFTER=3
+#$PCP_COMPRESSAFTER=never
+    
+# === LOGGER CONTROL SPECIFICATIONS ===
+#   
+#Host           P?  S?  directory                       args
+
+# local primary logger
+LOCALHOSTNAME   y   n   PCP_ARCHIVE_DIR/LOCALHOSTNAME   -r -T24h10m -c config.dee
+fault -v 100Mb
+     </screen>
+     <note>
+      <title>Defaults point to local host</title>
+      <para>
+       If you run the <command>pmlogger</command> in a container on a different
+       machine than runs the <command>pmcd</command> (a client), change the
+       following line to point to the client:
+      </para>
+<screen># local primary logger
+<replaceable>CLIENT_HOSTNAME</replaceable>   y   n   PCP_ARCHIVE_DIR/<replaceable>CLIENT_HOSTNAME</replaceable>   -r -T24h10m -c config.default -v 100Mb</screen>
+      <para>
+       For example, for the <literal>slemicro_1</literal> host name, the line
+       should look as follows:
+      </para>
+<screen># local primary logger
+slemicro_1   y   n   PCP_ARCHIVE_DIR/slemicro_1   -r -T24h10m -c config.default -v 100Mb</screen>
+     </note>
+    </section>
+    <section xml:id="sec-config-default">
```

Comment:
I would remove this section. I'm not exactly sure how you would add metrics after the auto generation is done on container start. Anyway I don't see the need to manually add metrics.